### ISSUE TITLE
Add "Open All" button to settings page to open all tabs

### ISSUE THREAD

**Issue by peepopoggers** [FreeTubeApp/FreeTube#1743]
**Is your feature request related to a usage problem (not a bug)? Please describe.**
It would make it much easier to look through the settings for a miscellaneous button if there was an easy way to open all of the tabs. It could float to the side of the settings tab, or could have a similar location.

**Describe the solution you'd like to see implemented**
To add a button to open all settings tabs at once

**Describe alternatives you've considered**
Opening them manually (takes 10s)

**Screenshot (mockup)**
![image](https://user-images.githubusercontent.com/72892531/134538281-73d68c75-cf68-48ba-bac6-00250bdbd8ed.png)

**Comment by jasonhenriquez** [comment]
Good idea -- could probably use a corresponding "Close All" as well. I'm still thinking about which control or controls would be best for this.

**Comment by peepopoggers** [comment]
@jasonhenriquez It could just be a toggle.
